examples/collapsibleWidgets.py

```diff
@@ -0,0 +1,29 @@
+import sys
```

Comment:
Great to have a simple sample code!

Can you add it to the [sample code gallery](https://www.silx.org/doc/silx/latest/sample_code/index.html)?

-> https://raw.githubusercontent.com/silx-kit/silx/refs/heads/main/doc/source/sample_code/index.rst

Reply:
<img width="720" height="525" alt="image" src="https://github.com/user-attachments/assets/4850ba7c-82bc-4de0-b73f-db7dce5c8278" />